src/components/About/Team/index.js

```diff
@@ -133,26 +133,26 @@ const Description = styled.div`
 
 const Team = props => {
   let [contributors, updataContributors] = useState([]);
-  /*eslint-disable */
 
-  useEffect(async () => {
-    // Adding title tag to page
+  useEffect(() => {
     document.title =
       'Developer Team of SUSI.AI - Open Source Artificial Intelligence for Personal Assistants, Robots, Help Desks and Chatbots';
 
-    let data = await getContributors();
-    let flags = [],
-      output = [];
-    for (let i = 0; i < data.length; i++) {
-      if (flags[data[i].name]) {
-        continue;
+    async function fetchData() {
```

Comment:
here `IIFE` can be used

Reply:
Using this also have the same effect as of IIFE, for using async function in hooks this is the recommended way. We can use either of them. Now it solves the problem of without having the need to reload the page.